core/services/workflows/metering/metering_test.go

```diff
@@ -697,32 +706,39 @@ func Test_Report_EmitReceipt(t *testing.T) {
 func Test_Report_CreditToSpendingLimits(t *testing.T) {
```

Comment:
Needs a test for SpendType<>Config the two cases (length mismatch & missing ratio) flipping on metering mode